app/controllers/articles_controller.rb

```diff
@@ -3,18 +3,15 @@ class ArticlesController < ApplicationController
   before_action :require_user, only: [:new, :edit, :create, :update, :destroy]
   before_action :require_same_user, only: [:edit, :update, :destroy]
 
-  def show
-  end
-
   def index
     @articles = Article.paginate(page: params[:page], per_page: 5)
   end
 
-  def new
-    @article = Article.new
+  def show
```

Comment:
Die Änderungen hier haben eigentlich nichts mit dem Ticket zu tun, oder?

Reply:
Nein, nicht direkt. Ich wollte die actions in eine Reihenfolge bringen, die sich für mich logischer anfühlte.